.github/workflows/java-pr.yml

```diff
@@ -27,8 +27,6 @@ on:
       # This will make it easier to verify action changes don't break anything.
       - '.github/actions/setup-env/*'
       - '.github/workflows/java-pr.yml'
-  schedule:
```

Comment:
same as comments of #1475, generally we would like to have same workflow run for different triggers. Any reason this didn't work?